system/jlib/jfile.cpp

```diff
@@ -5248,6 +5248,46 @@ bool isRemotePath(const char *path)
     }
 }
 
+bool containsRelPaths(const char *path)
+{
+    if (isEmptyString(path))
+        return false;
+    if (*path == '~')
+        return true;
+
+    const char *cur = path;
+    char sepChar = getPathSepCharEx(path);
+    char previousChar = sepChar;
+    for (;;)
+    {
+        switch (*cur)
+        {
+        case '.':
+        {
+            if (previousChar == sepChar)
+            {
+                cur++;
+                if ((*cur == sepChar) || (*cur == '\0')) // '.'
```

Comment:
@jakesmith should this be disallowed?  @GordonSmith could this cause any problems?
It means \var\lib\HPCCSystems\myhthor\.\hthor.log
will be disallowed.  Could there be any existing code where .\hthor.log is passed in as a relative path?  (Not just for logs but any of these affected functions.)